### DISCUSSION TITLE
iconについて質問です！

### DISCUSSION THREAD

**Discussion by KOTARO1209** [lit-iphone-course/lit-iphone-course#102] [Question-original]
https://github.com/KOTARO1209/icon

こちらアイコンが反映されないので問題をお聞きしたいです！

**Comment by txxxxc** [comment]
以下のフォーマットに従って質問してください！


[✅✅ 質問テンプレート ✅✅ · Discussion #18 · lit-iphone-course/lit-iphone-course](https://github.com/lit-iphone-course/lit-iphone-course/discussions/18)

**Reply by KOTARO1209** [reply]
申し訳ないです🙇

## (1) 実現したいこと
アイコンを反映したいです！
## (2) 実現するために自分が試した内容とその結果
教科書通りにやりました！
## (3) 調べた内容や資料
教科書「icon」
## (4) 原因だと思われそうな箇所(ソースコードをコピペして貼る)
不明です
## (5) GitHubのリンク
https://github.com/KOTARO1209/icon

**Reply by txxxxc** [reply]
>https://github.com/KOTARO1209/icon

リポジトリにicon.pngが入ってないっぽいけど、プロジェクトにicon.pngを適切に入れられているか確認してみてください！
